python/ray/tune/examples/mnist_pytorch_lightning.py

```diff
@@ -0,0 +1,252 @@
+# flake8: noqa
```

Comment:
don't forget to put this file into CI

see https://github.com/ray-project/ray/pull/7960/files